frontend/packages/knative-plugin/console-extensions.json

```diff
@@ -33,7 +33,25 @@
       "href": "/channel/ns/:namespace",
       "label": "%knative-plugin~Channel%",
       "description": "%knative-plugin~Create a Knative Channel to create an event forwarding and persistence layer with in-memory and reliable implementations%",
-      "icon": { "$codeRef": "icons.channelIconSVG" }
+      "icon": { "$codeRef": "icons.channelIconSVG" },
+      "accessReview": [
+        { "group": "eventing.knative.dev", "resource": "channels", "verb": "create" }
```

Comment:
FYI, Here I have added accessReview to `Serverless` cards, so it will not be shown if the user doesn't have create access
![image](https://user-images.githubusercontent.com/9964343/124231805-68503e80-db2e-11eb-9f2d-141d8b5466b7.png)

Reply:
@karthikjeeyar shall we add flags as well

```
"flags": {
      "required": ["KNATIVE_EVENTING", "FLAG_KNATIVE_EVENTING_CHANNEL"]
    }
```

Reply:
This flags are already there.